### PR TITLE
test: Fix failing ControllerEngineTests on AMD Ryzen 4800U

### PR DESCRIPTION
**Pull request by Holzhaus**
Related Zulip Discussion: https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/ControllerEngineTest.20failing

### CODE REVIEW

**Comment by Holzhaus** [comment]
@hacksdump does this fix the failing tests with CTest on your machine as well?

**Comment by Be-ing** [comment]
All tests pass locally for me.

**Comment by hacksdump** [comment]
> @hacksdump does this fix the failing tests with CTest on your machine as well?

Great. `ctest` works now.

File: src/test/controllerengine_test.cpp [hunk 1]
```diff
@@ -36,6 +36,16 @@ class ControllerEngineTest : public MixxxTest {
                                         QScriptValueList());
     }
 
+    void processEvents() {
+        // QCoreApplication::processEvents() only processes events that were
+        // queued when the method was called. Hence, events that were emitted
+        // by a slot that was triggered by processing are *not* processed and
```
Comment (hacksdump) [inline src/test/controllerengine_test.cpp:42]: Do we use `*not*` or `NOT` to highlight text in comments? Asking for future reference.

Comment (Holzhaus) [reply src/test/controllerengine_test.cpp:42]: No idea. Enclosing a string in `*` denotes an emphasis in markdown, and when generating doxygen documentation, the string will by displayed in italics.

File: src/test/controllerengine_test.cpp [hunk 2]
```diff
@@ -36,6 +36,16 @@ class ControllerEngineTest : public MixxxTest {
                                         QScriptValueList());
     }
 
+    void processEvents() {
+        // QCoreApplication::processEvents() only processes events that were
+        // queued when the method was called. Hence, events that were emitted
```
Comment (uklotzde) [inline src/test/controllerengine_test.cpp:41]: "Hence, all subsequent events that are emitted while processing those queued events will not be processed and are enqueued for the next event processing cycle. Calling processEvents() twice ensures that at least all queued *and* the next round of emitted events are processed."

**Comment by Holzhaus** [comment]
Merge?

**Comment by uklotzde** [comment]
Thank you! LGTM

**Pull request merged**